recipes/dbus/1.x.x/conanfile.py

```diff
@@ -63,6 +63,7 @@ def _configure_autotools(self):
             args.append("--disable-doxygen-docs")
             args.append("--disable-xml-docs")
 
+            args.append("--with-x=%s" % ("yes" if self.options.get_safe("with_x11", False) else "no"))
```

Comment:
`--with-x` or `--with-x11`? 👁️

Reply:
The Autotools switch is --with-x
(So it can be disabled using either --with-x=no or --without-x)

From ./configure --help
`  --with-PACKAGE[=ARG]    use PACKAGE [ARG=yes]`
`  --without-PACKAGE       do not use PACKAGE (same as --with-PACKAGE=no)`
`  --with-x                use the X Window System`

Reply:
https://github.com/freedesktop/dbus/blob/9b426c7916a98d24e61e6bbbe6c466e4e9b02fef/configure.ac#L1103-L1109